agent.py

```diff
@@ -422,7 +412,8 @@ def doTimestep(self, timestep):
             self.lastMoved = self.timestep
             self.doUniversalIncome()
             self.moveToBestCell()
```

Comment:
The old `updateNeighbors()` updated both `vonNeumannNeighbors` and `mooreNeighbors`, and then called `updateSocialNetwork()`. I've changed it to update `adjacentNeighbors` based on the structure of `adjacentCells` (which will either be Moore or von Neumann neighborhoods set at the beginning of the simulation). `updateSocialNetwork()` has been moved outside this function as its own part of `doTimestep()`.